modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+= Adding the {product-title} service
+Add the {product-title} RPM package to a blueprint and enable the {product-title} service to build an {op-system-ostree} image with {product-title}.
+
+.Image Builder blueprint example:
+
+[source,terminal]
```

Comment:
See my comment above on the blueprint file format. I suspect that [source,terminal] is not the right format.

Reply:
Is `[source,toml] the correct format for blueprint files?

Reply:
It resolves; couldn't find the reference anywhere in the asciidoctor docs.